### PR TITLE
Add TCMMF411.config

### PR DESCRIPTION
**Pull request by ThanYangFPV**
This is a new start. 
Please ignore pull requests that were wrong before.
Please do not add flags that are duplicate.
This configuration is for AIO use only.Dshot600 is enabled by default.

thank you.

### CODE REVIEW

File: configs/default/TCMMF411.config [hunk 1]
```diff
@@ -0,0 +1,110 @@
+# Betaflight / STM32F411 (S411) 4.1.1 Nov 15 2019 / 12:55:17 (1e5e3d369) MSP API: 1.42
+
+board_name TCMMF411
```
Comment (mikeller) [inline configs/default/TCMMF411.config:3]: The file name for this is wrong - please read and follow the [instructions](https://github.com/betaflight/betaflight/blob/master/docs/TargetMaintenance/CreatingAUnifiedTarget.md).

File: configs/default/TCMMF411.config [hunk 2]
```diff
@@ -0,0 +1,110 @@
...
+# pin B10: DMA1 Stream 1 Channel 3
+dma pin A08 0
+# pin A08: DMA2 Stream 6 Channel 0
+
+# feature
+feature -RX_PARALLEL_PWM
+feature RX_SERIAL
+feature OSD
+feature LED_STRIP
+feature AIRMODE
```
Comment (mikeller) [inline configs/default/TCMMF411.config:80]: Please follow the instructions - explicitly adding default settings is not acceptable.

**Review by mikeller — CHANGES_REQUESTED** [review]

File: configs/default/TCMMF411.config (hunk 1 above)
Comment (ThanYangFPV) [reply configs/default/TCMMF411.config]: OK.

File: configs/default/TCMMF411.config (hunk 2 above)
Comment (ThanYangFPV) [reply configs/default/TCMMF411.config]: OK,

File: configs/default/TCMM-TCMMF411.config [hunk 3]
```diff
@@ -0,0 +1,109 @@
...
+led 0 0,0::A:2
+led 1 1,0::A:2
+led 2 2,0::A:2
+led 3 3,0::A:2
+
+# master
+set baro_spi_device = 2
+set serialrx_provider = SBUS
+set blackbox_device = SPIFLASH
+set dshot_idle_value = 450
```
Comment (etracer65) [inline configs/default/TCMM-TCMMF411.config:94]: This is a very bad idea. Setting the idle too low can cause motor de-syncs and serious support issues.

Comment (ThanYangFPV) [reply configs/default/TCMM-TCMMF411.config]: Good suggestion, use the default idle value.

**Review by mikeller — APPROVED** [review]

**Pull request merged**